lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -14,15 +15,12 @@
 }
 
 .FilePickerApp__heading {
-  grid-column: 1 / 3;
-  margin-bottom: 1em;
+  grid-column: 1 / span 2;
 }
 
 .FilePickerApp__left-col {
   grid-column: 1;
-  padding-right: 20px;
-
-  font-weight: bold;
+  font-weight: 800;
```

Comment:
What was the rationale for this change? (I don't have an opinion on it, just curious)

Reply:
Pedantic. Just more adherent to our current conventions such as they are.